e2e-tests/test/evm-apis.test.ts

```diff
@@ -130,3 +130,22 @@ describe("evm_revert", function () {
     expect(reverted).to.be.true;
   });
 });
+
+describe("evm_setBlockGasLimit", function () {
+  it("Should change block gas limit", async function () {
+    // Arrange
+    const wallet = new Wallet(RichAccounts[0].PrivateKey, provider);
+
+    // Act
+    const blockBefore = await provider.getBlock("latest");
+    const limitBefore = BigNumber.from(blockBefore.gasLimit).toString();
+    await provider.send("evm_setBlockGasLimit", ["0x1000000000000"]);
+    await provider.send("evm_mine");
```

Comment:
```suggestion
    await provider.send("evm_mine");
    await provider.send("evm_mine");
```
then we will know that block gas limit is not set back to default  after 1 block